src/frontend/web_application/src/components/ContactAvatarLetter/index.jsx

```diff
@@ -29,10 +29,12 @@ const ContactAvatarLetter = ({ contact, size, isRound }) => {
 
   const letterClassNameSize = classNameSize ? `${classNameSize}__letter` : null;
 
+  const letterClassName = className ? `${className}__letter` : null;
```

Comment:
it is hard for me to solve the problem of _css only_ responsive ContactAvatarLetter without changing its className. Maybe one solution could be to have a fixed font-size for ContactAvatarLetter + use of flexbox to vertical align, instead of line-height.